models/perpetual/optimism/perpetual_v2_optimism_trades.sql

```diff
@@ -0,0 +1,82 @@
+{{ config(
+	schema = 'perpetual_v2_optimism',
+	alias ='trades',
+	partition_by = ['block_date'],
+	materialized = 'incremental',
+	file_format = 'delta',
+	incremental_strategy = 'merge',
+	unique_key = ['block_time', 'project', 'version', 'tx_hash', 'evt_index']
+	)
+}}
+
+{% set project_start_date = '2021-11-22' %}
+
+WITH perps AS (
+	SELECT
+		p.evt_block_time AS block_time
+		,"" AS virtual_asset
```

Comment:
as to @soispoke comment on missing column, i think it was an accidental comment on the wrong file. it's actually this script with the issue. notice here i applied force empty spaces for three columns, which all didn't exist but were called downstream